clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java

```diff
@@ -124,9 +124,9 @@ public static TachyonFile createByteFile(TachyonFileSystem tfs, String fileName,
     List<FileInfo> infos = tfs.listStatus(tfs.open(new TachyonURI(path)));
     List<String> res = new ArrayList<String>();
     for (FileInfo info : infos) {
-      res.add(info.getPath());
-
-      if (info.isFolder) {
+      if (!info.isFolder) {
+        res.add(info.getPath());
+      } else {
         res.addAll(listFiles(tfs, info.getPath()));
       }
```

Comment:
Apart from `MasterFaultToleranceIntegrationTest`, the only usage of this method is in `UfsUtilsIntegrationTest#loadUnderFsTest` were the return value of `listFiles` is checked to be null or not, so actually, its behavior only has effect on `MasterFaultToleranceIntegrationTest`.